lib/routes/devices-sessions.js

```diff
@@ -98,9 +99,16 @@ module.exports = (log, db, config, customs, push, devices) => {
           payload.id = sessionToken.deviceId
         }
 
-        if (payload.pushCallback && (! payload.pushPublicKey || ! payload.pushAuthKey)) {
-          payload.pushPublicKey = ''
-          payload.pushAuthKey = ''
+        const pushEndpointOk = ! payload.id || // New device.
+                               payload.id && payload.pushCallback // Updating the pushCallback
```

Comment:
If we do end up leaving the old value of `pushCallback` intact in the db, we'll have to update this check to assert that the client has provided an updated value.

Reply:
Is that ok to use the sessionToken value here?

Reply:
Using the sessionToken value seems OK